client/implementation/src/main/java/io/smallrye/graphql/client/impl/typesafe/reflection/TypeInfo.java

```diff
@@ -30,17 +32,37 @@
 import java.util.stream.Stream;
 import java.util.stream.Stream.Builder;
 
+import jakarta.json.bind.annotation.JsonbSubtype;
 import jakarta.json.bind.annotation.JsonbTransient;
+import jakarta.json.bind.annotation.JsonbTypeInfo;
 
 import org.eclipse.microprofile.graphql.Ignore;
 import org.eclipse.microprofile.graphql.NonNull;
 
+import io.smallrye.graphql.api.Union;
+import io.smallrye.graphql.client.impl.SmallRyeGraphQLClientMessages;
 import io.smallrye.graphql.client.typesafe.api.ErrorOr;
 import io.smallrye.graphql.client.typesafe.api.TypesafeResponse;
 import io.smallrye.mutiny.Multi;
 import io.smallrye.mutiny.Uni;
 
 public class TypeInfo {
+    private static final Class<? extends Annotation> JACKSON_JSON_IGNORE = findAnnotation(
+            "com.fasterxml.jackson.annotation.JsonIgnore");
+    private static final Class<? extends Annotation> JAKARTA_NOT_NULL = findAnnotation(
+            "jakarta.validation.constraints.NotNull");
+
+    private static Class<? extends Annotation> findAnnotation(String className) {
+        try {
+            //noinspection unchecked
+            return (Class<? extends Annotation>) Class.forName(className, false,
+                    Thread.currentThread().getContextClassLoader());
+        } catch (ClassNotFoundException e) {
+            /* IN CASE THE CLASS IS NOT IMPORTED */
+            return null;
+        }
+    }
+
```

Comment:
Actually off-topic, but we can do this stuff only once and not clutter the calling code.